maven-plugin/src/main/java/org/openmrs/maven/plugins/Run.java

```diff
@@ -174,6 +174,10 @@ private void runInFork(Server server) throws MojoExecutionException, MojoFailure
 					" -javaagent:\"" + Server.getServersPath().resolve("springloaded.jar").toAbsolutePath() + "\" -noverify";
 		}
 
+		if(Integer.parseInt(System.getProperty("java.version").split("\\.")[0]) >= 17) {
```

Comment:
I do not find this to be the ideal solution because there are cases where one needs to add more than these two. Just like you did for the appframework module.

Reply:
I am creating another PR to add a parameter that allows users to pass JVM arguments. However, I suggest that we should keep this as without this users won't be able to run the reference application using Java 17.

Reply:
Shouldn't we instead fix that reference application modules that require it?

Reply:
I agree. What about keeping this until we fix the reference application modules ? We could remove this after that.

Reply:
After adding it, what should one do for a module which requires an extra `--add-open`?

Reply:
Let  us start with the parameter passing capability.